### PR TITLE
Create and remove books

### PR DESCRIPTION
**Pull request by akucintavalent**
Implemented adding and removing books by using Redux Store.

### CODE REVIEW

File: src/components/add_book/AddBook.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    const newBook = {
+      id: uuidv4(), // make sure it's unique
+      title: titleElement.value,
+      author: authorElement.value,
+      category: categoriesElement.value,
+    };
+    authorElement.value = '';
+    titleElement.value = '';
+    dispatch(addBook(newBook));
+  };
```
Comment (omar-labana) [inline src/components/add_book/AddBook.js]: - [ ] The form responsible for adding a book should be a controlled component. The component should contain: - A local state with `title`, `author` - they should have their inputs that updates the local state every time they update (`onChange`) then dispatches the `addBook` action with the object containing a new id as its argument

**Review by omar-labana — CHANGES_REQUESTED** [review]
Hi @akucintavalent, 

Excellent work so far! 👌
You have implemented all the required features correctly and without any errors. ✔️ 
There are some cases that you need to adjust but you are almost done! 😀

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/add_book/AddBook.js (hunk 1 above)
Comment (akucintavalent) [reply src/components/add_book/AddBook.js]: What do you mean by "the form should be a controlled component". It is not very clear to me, sorry.

Comment (omar-labana) [reply src/components/add_book/AddBook.js]: @akucintavalent I tried reaching out on slack, anyway please check this https://stackoverflow.com/questions/42522515/what-are-react-controlled-components-and-uncontrolled-components#:~:text=A%20controlled%20component%20is%20a,kept%20in%20the%20component's%20state.

**Comment by omar-labana** [comment]
Hi @akucintavalent, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**